bluzelled/src/main.cpp

```diff
@@ -1,54 +1,39 @@
 #include "Node.h"
 #include "NodeUtilities.h"
 #include "web_sockets/WebSocketServer.h"
-#include "web_sockets/WebSocket.h"
-#include "web_sockets/Listener.h"
-#include "web_sockets/Session.h"
-#include "server/server.hpp"
+#include "http/server/ServiceManager.h"
 #include "NodeManager.h"
 
 #include <boost/exception/all.hpp>
-#include <boost/beast/core.hpp>
 #include <boost/beast/http.hpp>
-#include <boost/beast/version.hpp>
-#include <boost/asio/ip/tcp.hpp>
-#include <boost/asio/strand.hpp>
-#include <boost/config.hpp>
-#include <iostream>
-#include <string>
 
 static unsigned long s_max_nodes = 25;
 static unsigned long s_min_nodes = 5;
 static Nodes s_nodes;
+
 static boost::mutex *s_mutex = nullptr;
 
 // List of nodes that were died and need to be reported to GUI.
 static Nodes s_removed_nodes;
 static boost::mutex s_removed_nodes_mutex;
-Nodes* get_removed_nodes() { return &s_removed_nodes; }
-boost::mutex& get_removed_nodes_mutex() { return s_removed_nodes_mutex; }
 
 std::shared_ptr<Listener> g_listener; // Listener object from WebSocketServer.
 
-
-void print_message(const std::string &msg);
-
-void kill_and_join_all_nodes(Nodes &nodes);
-
-void make_introductions(const Nodes &nodes);
-
 boost::mutex &get_mutex();
 
-Nodes create_nodes(int number_of_nodes, std::shared_ptr<Listener> l);
-
-
-void add_nodes(const Nodes &nodes)
+void add_nodes
+        (
+                const Nodes &nodes
+        )
 {
     boost::mutex::scoped_lock lock(*s_mutex);
     s_nodes.insert(s_nodes.end(), nodes.begin(), nodes.end());
 }
 
-void set_max_nodes(unsigned long max)
+void set_max_nodes
+        (
+                unsigned long max
+        )
 {
```

Comment:
@rich If I use reformat code feature from CLIon it always places curly brackets on the sam eline right after method name. In your case Node.h is like mine but min.cpp is different. I suggest running "Reformat Code" to set them properly.